html/cross-origin-opener-policy/coep-navigate-popup.https.html

```diff
@@ -43,8 +46,14 @@
     "coep": "",
     "opener": false
   }
-].forEach(variant => {
+].forEach((variant, i) => {
   ["same-origin", "same-site"].forEach(site => {
+    // Only run specified variants
+    if (!window.location.search.substr(1).split(",").includes(i.toString())) {
```

Comment:
I think I prefer using `subset-test.js` and `if (!shouldRunSubtest(i)) { return; }`

Although this is less code, it's missing some aspects like ability to run all tests when not using a query string and the "-last" thing which helps somewhat mitigate the risk of forgetting to update `<meta>`s when adding a new test.

Reply:
Mkay.

Reply:
I don't understand why that utility suggests 1-based indexing or how that even works given the code.

Also, given that I'm only doing this for the inner forEach loop there's a 2x difference with what I want. Sigh.

Reply:
The first test is 1 when using `subsetTest()`. Using `shouldRunSubTest()` manually without `subsetTest()` it's up to you which numbers you use.

I don't follow about the 2x difference. Can you clarify?

Reply:
Oh, I see. Can't you move it to the outer loop?

Reply:
If I don't care about generating variants this is fine, right?

Reply:
Outer loop would have the same problem btw, you'd need a counter across loops.

Reply:
Oh, but I could move the check, yes.